src/v0/__tests__/index.spec.ts

```diff
@@ -0,0 +1,36 @@
+import fetchMock from 'fetch-mock';
+import { endpoint } from '../analytics';
+import connection from '../index';
+
+describe('connection', () => {
+  describe('analytics', () => {
+    afterEach(fetchMock.restore);
+
+    it('sends an analytics component load event for a component on initialization', async () => {
+      const componentVersion = 'v0.0.0';
+      fetchMock.mock(endpoint.local, {});
+      connection({
+        env: 'local',
+        componentVersion,
+        element: document.createElement('manifold-product'),
+        getAuthToken: () => '',
+        getOwnerId: () => '',
+        clearAuthToken: () => {},
+        clientId: '123',
+      });
+      expect(fetchMock.called(endpoint.local)).toBe(true);
+      const res = fetchMock.calls()[0][1];
+      expect(res && res.body && JSON.parse(res.body.toString())).toEqual({
+        source: 'MANIFOLD-PRODUCT',
```

Comment:
Why is this the source?

Reply:
Oh oh it’s just a test nvm

Reply:
ya well its also a good question on the source v. component.....

Source being the higher level namespace and the component being the actual HTMLComponent. Philisophically maybe that isn't really useful without composable components.

I can see us having multiple composable components in the future such as MANIFOLD-PLAN which would be the component and the source could be a parent such as MANIFOLD-PRODUCT or something... for now components source and component are the same.

Reply:
Yeah that makes sense. We can always adjust analytics and get more granular as-needed for sure (obviously you want to do that as soon as you realize something but also you probably won’t get it 100% correct first pass either).